src/kvstore/hbase/HBaseStore.h

```diff
@@ -0,0 +1,128 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef KVSTORE_HBASESTORE_H_
+#define KVSTORE_HBASESTORE_H_
+
+#include "base/Base.h"
+#include "kvstore/KVStore.h"
+#include "kvstore/PartManager.h"
+#include "kvstore/KVEngine.h"
+#include <gtest/gtest_prod.h>
+
+namespace nebula {
+namespace kvstore {
+
+struct GraphSpaceHBase {
+    Engine engine_;
+    std::set<PartitionID> parts_;
+};
+
+
+class HBaseStore : public KVStore, public Handler {
```

Comment:
HBaseStore should not derived from Handler.

Reply:
Done!